torch2jax/__init__.py

```diff
@@ -522,9 +534,7 @@ def sum(input, dim=None, keepdim=False, dtype=None):
 @implements(torch.tensor)
 def tensor(data, dtype=None, device=None, requires_grad=False, pin_memory=False):
   assert not requires_grad
-  return jnp.array(
```

Comment:
We should preserve `None` as it is so that `jax` will infer the type from the data itself instead of `torch.get_default_dtype()`. The latter will make `torch.tensor(1)` a float tensor instead of integer.